isofit/radiative_transfer/engines/six_s.py

```diff
@@ -447,3 +436,51 @@ def parse_file(file, wl, multipart_transmittance=False, wl_size=0) -> dict:
         data.pop("grid")
 
         return data
+
+
+def get_exe(path: str = None, version: bool = False) -> str:
+    """
+    Retrieves the 6S executable from a given path
+
+    Parameters
+    ----------
+    path : str, default=None
+        6S directory path. If None, defaults to the ini sixs path
+    version : bool, default=False
+        Returns the 6S version instead
+
+    Returns
+    -------
+    pathlib.Path | str
+        Either the 6S executable as a pathlib object or the string 6S version
+    """
+    if path is None:
+        path = env.sixs
+
+    path = Path(path)
+
+    exes = path.glob("sixsV*")
+    exes = [exe for exe in exes if "lutaero" not in exe.name]
+    names = [exe.name for exe in exes]
+
+    if not exes:
+        raise FileNotFoundError(f"Could not find a 6S executable under path: {path}")
+
+    if len(exes) > 1:
+        Logger.warning(
+            f"More than one 6S executable was found. Defaulting to the first one: {names}"
+        )
+
+    if version:
+        # Try using the version.txt file, created by the isofit downloader
+        if (txt := path / "version.txt").exists():
+            with txt.open("r") as f:
+                vers = f.read()
+        # Fallback to using the executable name
+        else:
+            _, vers = names[0].split("V")
+            vers = f"v{vers}".lower()
```

Comment:
Do we want this condition to also get hit of the version boolean is false?  Strikes me that we may.  Thoughts?